QEfficient/utils/_utils.py

```diff
@@ -119,13 +119,13 @@ def onnx_exists(model_name: str) -> Tuple[bool, str, str]:
     return onnx_exists_bool, onnx_dir_path, onnx_model_path
 
 
-def load_hf_tokenizer(model_name: str, cache_dir: Optional[str] = None, hf_token: Optional[str] = None, padding_side:str = "right", **kwargs) -> Union[PreTrainedTokenizerFast, PreTrainedTokenizer]:
-    logger.info(f"Loading Tokenizer for {model_name}")
+def load_hf_tokenizer(pretrained_model_name_or_path: str, cache_dir: Optional[str] = None, hf_token: Optional[str] = None, padding_side:str = "left", **kwargs) -> Union[PreTrainedTokenizerFast, PreTrainedTokenizer]:
```

Comment:
okay to keep default padding to "`right`"?

Reply:
will update, these changes came in, as I was reviving this old branch, and main was too ahead of this.